### PR TITLE
Dev > Master for release 1.5

### PR DESCRIPTION
**Pull request by ewels**
Merging the `dev` branch to `master` for a v1.5 release of `tools`.

### CODE REVIEW

**Comment by ewels** [comment]
This pull request **introduces 4 alerts** when merging 7049c8561fccd201b50196fdeba7b7967c58a387 into 54e233a4c167f515b6f616b3c5a8a9bd660861c0 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-eabe053d3ba2f2d572d42e6691a89bd507a853c3)

**new alerts:**

* 3 for Unused import
* 1 for Unused local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ewels** [comment]
This pull request **introduces 4 alerts** when merging c970d9b5c08013117143813e735a582364e896f6 into 54e233a4c167f515b6f616b3c5a8a9bd660861c0 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-0946cdf51934e4d56ead0af047535b9f35be1402)

**new alerts:**

* 3 for Unused import
* 1 for Unused local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ewels** [comment]
This pull request **introduces 2 alerts** when merging c89f5170fcc52d686764dbce20b33b878fb64115 into 54e233a4c167f515b6f616b3c5a8a9bd660861c0 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-ab88778e92b62a99fa09de2bf4422907f9f51b3e)

**new alerts:**

* 1 for Unused local variable
* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+* `nf-core list` now only shows a value for _"is local latest version"_ column if there is a local copy.
+* Lint markdown formatting in automated tests
+  * Added `markdownlint-cli` for checking Markdown syntax in pipelines and tools repo
+* Syncing now reads from a `blacklist.json` in order to exclude pipelines from being synced if necessary.
+* Added nf-core tools API description to assist developers with the classes and functions available.
+  * Docs are automatically built by Travis CI and updated on the nf-co.re website.
+* Introduced test for filtering remote workflows by keyword.
+* Build tools python API docs
+  * Use Travis job for api doc generation and publish
+* Bump `conda` to 4.5.12 in base nf-core Dockerfile
```
Comment (apeltzer) [inline CHANGELOG.md]:
```suggestion
* Bump `conda` to 4.5.12 in base nf-core Dockerfile
```

As listed in ''OTHER'' down further, we can drop this as we already updated to 4.6.X later 😆

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf [hunk 2]
```diff
@@ -350,3 +337,27 @@ workflow.onComplete {
     log.info "[{{ cookiecutter.name }}] Pipeline Complete"
 
 }
+
+
+def nfcoreHeader(){
```
Comment (apeltzer) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf]: AMAZING, just AMAZING.

Comment (ewels) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf]: badger

File: CHANGELOG.md (hunk 1 above)
Comment (ewels) [reply CHANGELOG.md]: Fixed 👍

**Comment by ewels** [comment]
This pull request **introduces 2 alerts** when merging f10f2fd6484d3a87073e36fba9c82d252108abec into 54e233a4c167f515b6f616b3c5a8a9bd660861c0 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-61ddce77cbe6c36dbf7fe807629a9cf21c562586)

**new alerts:**

* 1 for Unused local variable
* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**